File: web/src/main/resources/public/js/gift/guest/Guest.js
var Guest = function($guestForm) {
    this.name = $guestForm.find('.guest_name').val();
    this.weddingId = $guestForm.find('.weddingId').val();
    this.weddingUrl = $guestForm.find('.weddingUrl').val();
    this.phoneNumber = $guestForm.find('.phoneNumber').val();
    this.price = $guestForm.find('.price').val();
    this.message = $guestForm.find('.message').val();
    this.giftId = $guestForm.find('.gift_id').val();
}

File: core/src/main/java/com/seolgi/wedding/core/guest/Guest.java
package com.seolgi.wedding.core.guest;

import com.seolgi.wedding.core.gift.Gift;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.CreationTimestamp;

import javax.persistence.*;
import java.util.Date;

@Entity(name = "guest")
@Getter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Guest {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(length = 20 , nullable = false)
    private String phoneNumber;

    @Column(length = 50 , nullable = false)
    private String name;

    @Column(length = 1000 , nullable = false)
    private String message;

    @Column(length = 100)
    private String sessionId;

    @ManyToOne
    @JoinColumn(name = "giftId" , referencedColumnName = "id")
    private Gift gift;

    @Column
    private long price;

    @CreationTimestamp
    @Temporal(value = TemporalType.TIMESTAMP)
    @Column(updatable = false)
    private Date createDate;

    public void merge(Guest guest) {
        this.sessionId = guest.getSessionId();
    }
}
